lib/lambda_ethereum_consensus/p2p/peerbook.ex

```diff
@@ -81,14 +111,10 @@ defmodule LambdaEthereumConsensus.P2P.Peerbook do
   defp prune() do
```

Comment:
We can do something similar to the peer selection in the pruning and just prune the lower score ones but is probably something to tackle in a PR that focus on the peer-scoring algorithm.